css/theme-dark.css

```diff
@@ -1,5 +1,15 @@
 body[theme="dark"] .search-container {
   border-color: #C1C1C1;
+  background-color: var(--dark-background-color);
+}
+
+body[theme="dark"] #search-input,
+body[theme="dark"] .mdc-icon-button {
+  color: white;
```

Comment:
this should probably be var(--dark-accent-color)

Reply:
Got'cha, will fix.

Reply:
addressed in main thread conversation about overall colors